stream-chat-android/src/main/java/com/getstream/sdk/chat/coil/StreamImageLoaderFactory.kt

```diff
@@ -19,8 +21,19 @@ public class StreamImageLoaderFactory(
             .allowHardware(false)
             .crossfade(true)
             .okHttpClient {
+                val cacheControlInterceptor = Interceptor { chain ->
+                    chain.proceed(chain.request())
+                        .newBuilder()
+                        .header("Cache-Control", "max-age=31536000,public")
```

Comment:
> 31536000

This is one year, right? Isn't that a bit long for caching images?

Reply:
Agree, what's the reasonable time span? One hour?

Reply:
At the same time, all this config looks weird, we should allow customers to take control over image loading
chatUI.setImageLoader(StreamCoilImageLoader(clientsCoilInstance))
chatUI.setImageLoader(StreamGlideImageLoader(clientsGlideInstance))
chatUI.setImageLoader(StreamPicassoImageLoader(clientsPicassoInstance))
chatUI.setImageLoader(object: StreamImageLoader() { // completely custom image loader implemented by client  })

Reply:
Changed cache expiration time to 1 hour

Reply:
>we should allow customers to take control over image loading

That can go on the backlog 😄 

>Changed cache expiration time to 1 hour

Sounds good for a start, we can revisit it later if we have to